src/pages/components/table.mdx

```diff
@@ -193,6 +193,49 @@ import TableVirtualizedCode from '!!raw-loader!../../examples/table/TableVirtual
 
 ## API
 
+The Table component follows this structure:
+
+```jsx
+<Table>
+  <Caption />
+  <Head>
+    <HeaderRow>
+      <HeaderCell />
+      <HeaderCell />
+      <SortableCell /> {/* Optional */}
+      {/* etc. */}
+    </HeaderRow>
+  </Head>
+  <Body>
+    <GroupRow>
+      <Cell />
+    </GroupRow>
```

Comment:
Should this be marked optional? Or is it really elemental to the canonical structure?